.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseIndianNumberParser.cs

```diff
@@ -0,0 +1,838 @@
+                                if (Config.ResolveCompositeNumber(fracWords[splitIndex]) >= hundredsSM &&
+                                    !Config.WrittenFractionSeparatorTexts.Contains(fracWords[splitIndex + 1]) &&
+                                    Config.ResolveCompositeNumber(fracWords[splitIndex + 1]) < hundredsSM)
+                                {
+                                    splitIndex++;
+                                    break;
+                                }
+
+                                splitIndex++;
+                            }
+
+                            break;
+                        }
+
+                        continue;
+                    }
+
+                    splitIndex++;
+                    break;
+                }
+
+                if (splitIndex < 0)
+                {
+                    splitIndex = 0;
+                }
+
+                var fracPart = new List<string>();
+                for (var i = splitIndex; i < fracWords.Count; i++)
+                {
+                    if (fracWords[i].Contains("-"))
+                    {
+                        var split = fracWords[i].Split('-');
+                        fracPart.Add(split[0]);
+                        fracPart.Add("-");
+                        fracPart.Add(split[1]);
+                    }
+                    else
+                    {
+                        fracPart.Add(fracWords[i]);
+                    }
+                }
+
+                fracWords.RemoveRange(splitIndex, fracWords.Count - splitIndex);
+
+                // Split mixed number with fraction
+                var denominator = GetTheIntValue(fracPart);
+                double numerValue = 0;
+                double intValue = 0;
+
+                var mixedIndex = fracWords.Count;
+                for (var i = fracWords.Count - 1; i >= 0; i--)
+                {
+                    if (i < fracWords.Count - 1 && Config.WrittenFractionSeparatorTexts.Contains(fracWords[i]))
+                    {
+                        var numerStr = string.Join(" ", fracWords.GetRange(i + 1, fracWords.Count - 1 - i));
+                        numerValue = GetTheIntValue(GetMatches(numerStr));
+                        mixedIndex = i + 1;
+                        break;
+                    }
+                }
+
+                var intStr = string.Join(" ", fracWords.GetRange(0, mixedIndex));
+                intValue = GetTheIntValue(GetMatches(intStr));
+
+                // Find mixed number
+                if (mixedIndex != fracWords.Count && numerValue < denominator)
+                {
+                    result.Value = intValue + (numerValue / denominator);
+                }
+                else
+                {
+                    result.Value = (intValue + numerValue) / denominator;
+                }
+            }
+
+            return result;
+        }
+
+        protected ParseResult ParseDigitNumber(ExtractResult extResult)
```

Comment:
Similar comments for most method from here on. Some seem exactly the same as in the base class.

Reply:
Unfortunately I won't have time in the next couple days to do an exact comparison. Sorry if I missed something obvious.

Reply:
@tellarin: feedback noted, we will work on it. the main hindrance is that the said functions in base have a protection level that does not allow them to be overridden, hence why we included them as they are needed. The behavior is not not that much different from the base but only some peculiarities are addressed in Indian refactored. We are not straying too far from the base to stick to base behavior.  We will review these one by one.